src/components/implementation/no_interface/llbooter/llbooter.c

```diff
@@ -299,6 +320,64 @@ comps_init(void)
 	return;
 }
 
+void 
+chkpt_comp_init(struct crt_comp *comp, struct crt_chkpt *chkpt, char *name)
+{
+#ifdef ENABLE_CHKPT
+	/* create the component */
+	void *elf_hdr;
+	int   keylen;
+	compid_t id = crt_ncomp() + 1;
```

Comment:
comment:

`/* We only support a single checkpoint directly above the existing components */`